UhooiPicBookWidgetsConfigurableIntent/IntentHandler.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  IntentHandler.swift
+//  UhooiPicBookSelectIntent
+//
+//  Created by Takehito Koshimizu on 2020/11/14.
+//
+
+import Intents
+import FirebaseCore
+
+final class IntentHandler: INExtension, ConfigurationIntentHandling {
+
+    private let repository: MonstersRepository
+
+    override init() {
+        FirebaseApp.configure()
+        repository = MonstersFirebaseClient()
+        super.init()
+    }
+
+    override func handler(for intent: INIntent) -> Any { self }
+
+    func provideMonsterOptionsCollection(
+        for intent: ConfigurationIntent,
+        with completion: @escaping (INObjectCollection<INMonster>?, Error?) -> Void) {
+
+        repository.loadMonsters { result in
+            switch result.map([INMonster].orderedMonsters) {
+            case let .success(items):
+                completion(INObjectCollection(items: items), nil)
+
```

Comment:
難しい処理をかんたんにさせていただきました 🙏 

```suggestion
        self.repository.loadMonsters { result in
            switch result {
            case let .success(monsters):
                let inMonsters = monsters
                    .sorted { $0.order < $1.order }
                    .map(INMonster.init)
                completion(INObjectCollection(items: inMonsters), nil)
```